test/suites/examples/create-eks.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: create-eks-task
+  namespace: tekton
+spec:
+  description: |
+    Create an EKS cluster and MNG.
+  params:
+  - name: cluster-name
+    description: The name of the EKS cluster you want to spin up.
+  - name: k8s-version
+    description: The K8s version used for the cluster.
+  - name: aws-region
+    description: The region where the cluster will be created.
+  steps:
+  - name: create-cluster
+    image: docker.io/weaveworks/eksctl:0.86.0
+    script: |
+      eksctl create cluster -f - << EOF
+      ---
+      apiVersion: eksctl.io/v1alpha5
+      kind: ClusterConfig
+      metadata:
+        name: $(params.cluster-name)
+        region: $(params.aws-region)
+        version: "$(params.k8s-version)"
+        tags:
+          karpenter.sh/discovery: $(params.cluster-name)
+      managedNodeGroups:
+        - instanceType: m5.large
```

Comment:
Should the instance type also be parametized?

Reply:
We could, but I don't think this matters as this is just the node for the Karpenter service pod to be installed. I think the value in parameterizing this is very low for how it makes the config larger.